proposals/dl_fit_class_mlp.json

```diff
@@ -0,0 +1,159 @@
+{
+    "id": "dl_fit_class_mlp",
+    "summary": "Train a Multilayer Perceptron classification model",
+    "description": "Fit a Multilayer Perceptron (MLP) classification model to training data. MLP is a class of feedforward artificial neural network (ANN) that consists of at least three layers of nodes: an input layer, a hidden layer, and an output layer. MLP utilizes a supervised learning technique called backpropagation for training.\n\nThis implementation is inspired by Z. Wang, W. Yan, and T. Oates (2017), Time Series Classification from Scratch with Deep Neural Networks: A Strong Baseline.",
```

Comment:
> This implementation is inspired by Z. Wang, W. Yan, and T. Oates (2017) ...

What is this reference about? And especially what doe "this implementation" refer to (I don't think there is already an implementation in openeo)?

Reply:
True , I have updated to reflect that and the paper is on the link section.